googledevices/cli/commands.py

```diff
@@ -0,0 +1,82 @@
+"""CLI commands."""
+import asyncio
+import json
+import click
+import aiohttp
+
+
+@click.group()
+def device_info_group():
+    """Click group."""
+    pass
+
+
+@click.group()
+def scan_network_group():
+    """Click group."""
+    pass
+
+
+@click.group()
+def controll_notifications_group():
+    """Click group."""
+    pass
+
+
+@click.group()
+def reboot_group():
+    """Click group."""
+    pass
+
+
+@device_info_group.command()
+@click.argument('ip_address', required=1)
+def device_info(ip_address):
+    """Get information about a Google device on your network."""
+    from googledevices.api.device_info import DeviceInfo
+
+    async def get_device_info():
+        """Get device info."""
+        async with aiohttp.ClientSession() as session:
+            googledevices = DeviceInfo(event_loop, session, ip_address)
+            await googledevices.get_device_info()
+            print(json.dumps(googledevices.device_info,
+                             indent=4, sort_keys=True))
+    event_loop = asyncio.get_event_loop()
```

Comment:
Maybe we can make a single event loop to run all the tasks inside

Reply:
Bad copy paste :P
Good catch! :)